lib/Theme.php

```diff
@@ -43,12 +43,12 @@ class Theme extends Core {
 	public $version;
 
 	/**
-	 * Timber\Theme object for the parent theme (if it exists), false otherwise
+	 * Timber\Theme object for the parent theme (even if the current theme is the parent theme)
```

Comment:
```suggestion
	 * Timber\Theme object for the parent theme.
	 *
	 * Always returns the top-most theme. If the current theme is also the parent theme, it will
	 * return itself.
```